pkg/controller/elasticsearch/remotecluster/remoteca/labels.go

```diff
@@ -0,0 +1,62 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package remoteca
+
+import (
+	"fmt"
+
+	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/label"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// RemoteClusterNamespaceLabelName used to represent the namespace of the RemoteCluster in a TrustRelationship.
+	RemoteClusterNamespaceLabelName = "remotecluster.k8s.elastic.co/namespace"
+	// RemoteClusterNameLabelName used to represent the name of the RemoteCluster in a TrustRelationship.
+	RemoteClusterNameLabelName = "remotecluster.k8s.elastic.co/name"
+	// TypeLabelValue is a type used to identify a Secret which contains the CA of a remote cluster.
+	TypeLabelValue = "remote-ca"
+	// certsInternalSecretName is the suffix added to the aforementioned Secret.
+	certsInternalSecretName = "remote-ca"
+)
+
+func remoteCAObjectMeta(
+	name string,
+	owner *esv1.Elasticsearch,
+	remote types.NamespacedName,
+) metav1.ObjectMeta {
+	return metav1.ObjectMeta{
+		Name:      name,
+		Namespace: owner.Namespace,
+		Labels: map[string]string{
+			RemoteClusterNamespaceLabelName: remote.Namespace,
+			RemoteClusterNameLabelName:      remote.Name,
+			label.ClusterNameLabelName:      owner.Name,
```

Comment:
is the CA in this secret the CA of the remote cluster, or the CA of the owner cluster?

Reply:
We copy the CA of the remote clusters, but the owner is the local one.